com.woltlab.wcf/option.xml

```diff
@@ -1683,5 +1733,11 @@ DESC:wcf.global.sortOrder.descending</selectoptions>
 	</import>
 	<delete>
 		<option name="module_attachment"/>
+		<option name="blacklist_sfs_username_time_period" />
```

Comment:
Are these part of the development process? Maybe they should be removed if they never were part of a publicly released version.

Reply:
I've removed them, I've gone through a lot of iterations during the development, especially testing out different approaches. That's also the reason there is no confidence rating, because it doesn't reflect the different reliability of the values and just using arbitrary multipliers doesn't cut it.